policies/github/repository.rego

```diff
@@ -716,4 +716,23 @@ repository_secret_is_stale[stale] := true{
         "update date" : time.format(secret.updated_at),
     }
 
+}
+
+# METADATA
+# scope: rule
+# title: Secret Scanning should be enabled for public repositories
+# description: Public repositories should not have secrets hard-coded in the files. It is recommended to enable some sort of secret scanning.
```

Comment:
Need to rephrase the policy desc, the issue applies for private repositories as well